Source/Core/ReferencePointGeometryInstanceAttribute.js

```diff
@@ -0,0 +1,150 @@
+define([
+        './Cartesian3',
+        './Cartographic',
+        './Check',
+        './ComponentDatatype',
+        './defined',
+        './defineProperties',
+        './EncodedCartesian3',
+        './Matrix4',
+        './Transforms'
+    ], function(
+        Cartesian3,
+        Cartographic,
+        Check,
+        ComponentDatatype,
+        defined,
+        defineProperties,
+        EncodedCartesian3,
+        Matrix4,
+        Transforms) {
+    'use strict';
+
+    /**
+     * Batch table attribute representing the HIGH or LOW bits of an EncodedCartesian3.
+     *
+     * @param {Cartesian3} vec3 HIGH or LOW bits of an EncodedCartesian3
+     * @private
+     */
+    function ReferencePointGeometryInstanceAttribute(vec3) {
```

Comment:
Maybe I will find out later, but what does "Reference Point" mean?

Should this name be better aligned with the `EncodedCartesian3` class?